src/states/infinitepeps.jl

```diff
@@ -188,6 +188,18 @@ Base.rotl90(A::InfinitePEPS) = InfinitePEPS(rotl90(rotl90.(unitcell(A))))
 Base.rotr90(A::InfinitePEPS) = InfinitePEPS(rotr90(rotr90.(unitcell(A))))
 Base.rot180(A::InfinitePEPS) = InfinitePEPS(rot180(rot180.(unitcell(A))))
 
+## Tensor-wise normalization
+
+function LinearAlgebra.normalize!(peps::InfinitePEPS, p::Real=2)
+    for idx in CartesianIndices(peps.A)
+        peps.A[idx] /= LinearAlgebra.norm(peps.A[idx], p)
+    end
```

Comment:
```suggestion
    normalize!.(unitcell(peps), p)
```
TensorKit should support `normalize` directly.

Reply:
After some thinking I decide not to define `LinearAlgebra.normalize`, but use `normalize!.(unitcell(peps), p)` directly. The former gives people the impression that one is normalizing the entire `peps` such that `<peps|peps> = 1`, but actually I'm just performing tensorwise normalization.